spec/support/schemas/claims_api/v2/errors/422.json

```diff
@@ -0,0 +1,25 @@
+{
+  "required": ["errors"],
+  "properties": {
+    "errors": {
+      "type": "array",
+      "items": {
+        "additionalProperties": false,
+        "required": ["status", "detail", "source"],
+        "properties": {
+          "status": {
+            "type": "integer",
+            "description": "HTTP error code"
+          },
+          "detail": {
+            "type": "string",
+            "description": "HTTP error detail"
+          },
+          "source": {
+            "type": "string"
+          }
+        }
+      }
+    }
+  }
+}
```

Comment:
this is the same as the `spec/support/schemas/claims_api/errors/default_with_source.json`, right? Could we not reuse that. Also, the LH standards show 

```
{
  "errors": [
    {
      "status": "422",
      "source": { "pointer": "/data/attributes/isRefillable" },
      "title": "Invalid Attribute",
      "detail": "isRefillable must be a boolean value."
    }
  ]
}
```

so I'd vote we add `title` in. https://department-of-veterans-affairs.github.io/lighthouse-api-standards/errors/#error-schemas

Reply:
I don't think we can reuse that since that is used in documentation for v1 so adding in 'title' would also change that documentation.

Reply:
@FonzMP With our error handling in the `disability_compensation_validations` file we are using `::Common::Exceptions::UnprocessableEntity` .  However that will never return a title as far as I can tell, it inherits from https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/common/exceptions/service_error.rb,  Would it be worth it either here in this ticket or another to update this (or get our own) so that we have a `title` in the returned error message?